metaworld/envs/mujoco/sawyer_xyz/sawyer_disassemble_peg.py

```diff
@@ -186,21 +186,7 @@ def _set_goal_marker(self, goal):
             goal[:3]
         )
 
-    def _set_obj_xyz_quat(self, pos, angle):
-        quat = Quaternion(axis = [0,0,1], angle = angle).elements
-        qpos = self.data.qpos.flat.copy()
-        qvel = self.data.qvel.flat.copy()
-        qpos[9:12] = pos.copy()
-        qpos[12:16] = quat.copy()
-        qvel[9:15] = 0
-        self.set_state(qpos, qvel)
-
-    def _set_obj_xyz(self, pos):
```

Comment:
Hmm I am wondering whether if we should remove this. I think we kinda have different `_set_obj_xyz` for different environments, especially that we have two tasks using two objects.

Reply:
But, I think it is good if we can remove this since these codes are copied a lot of times across this codebase

Reply:
Hi @zhanpenghe ! Thank you for the quick review. I used find-and-replace to refactor the functions, they were identical across a few dozen files. In my opinion, if some of the environments need a different implementation of _set_obj_xyz, they can override the function in the implementation.